### DISCUSSION TITLE
error setting certificate file

### DISCUSSION THREAD

**Discussion by RogerInHawaii** [community/community#89913] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm trying to clone the prusaslicer repository and immediately getting an error message:

"error setting certificate file"

My command within GIT CM is :

C:\GeminiSlicerVersions\GeminiSlicer2>git clone https://github.com/prusa3d/PrusaSlicer.git
Cloning into 'PrusaSlicer'...
fatal: unable to access 'https://github.com/prusa3d/PrusaSlicer.git/': error setting certificate file: C:/Program Files/Git/mingw64/etc/ssl/certs/ca-bundle.crt

I had successfully done GT cloning on a different computer but am now experiencing this certificate issue on my new computer.

Is there something I'm doing wrong, or has Prusa not kept its certificate up to date?

What can I do to resolve this?

**Comment by ayalamarcelo** [comment]
Also you can try with fork the repo and git clone https://github.com/[your-username]/PrusaSlicer.git  --depth=1.

**Comment by Mamadou-Saliou-Diallo** [comment]
error setting certificate file: C:\Program Files\Git\mingw64\ssl\cert.pem

**Comment by Mamadou-Saliou-Diallo** [comment]
S'il vous plait, aidez moi

**Comment by jfclere** [comment]
env GIT_SSL_NO_VERIFY=true git clone https://github.com/prusa3d/PrusaSlicer.git might help.
